packages/edit-site/src/components/editor/index.js

```diff
@@ -59,7 +58,7 @@ import TemplatePartConverter from '../template-part-converter';
 import { useSpecificEditorSettings } from '../block-editor/use-site-editor-settings';
 
 const { BlockRemovalWarningModal } = unlock( blockEditorPrivateApis );
-const { ExperimentalEditorProvider: EditorProvider } =
+const { ExperimentalEditorProvider: EditorProvider, ListViewSidebar } =
```

Comment:
It was already there, but I guess it's not needed to call it `ExperimentalEditorProvider` if it's a locked API?

Reply:
I think the reason is that we have a stable and a private one.